src/configs/settings.ts

```diff
@@ -1729,6 +1756,26 @@ export const irsReportingCongif: {
 };
 /* tslint:enable:object-literal-sort-keys */
 
+/** Thresholds lookup
+ * For custom indicator thresholds, define it and add it here,
+ * otherwise the default indicator thresholds will be used.
+ */
+export const indicatorThresholdsLookUpIRSLite: IndicatorThresholdsLookUp = {
+  irsLite2020: indicatorThresholdsIRSLite,
+};
+
+/* tslint:disable:object-literal-sort-keys */
+/** The actual configuration object controlling how IRS Reporting is handled for different clients */
+export const irsLiteReportingCongif: {
+  [key: string]: IrsReportingConfig;
+} = {
+  // Namibia Structures Configs
+  [process.env.REACT_APP_SUPERSET_IRS_LITE_REPORTING_STRUCTURES_DATA_SLICE_NA as string]: {
```

Comment:
Lets follow the convention we already have. 
Move the processing of envs to `env.ts` file and also provide a default value in cases were the env is not provided.

Reply:
This is following the conventions in the code. Look two methods higher to see the existing irsReportingConfig that this was coppied from.